Solutions/Tomcat/Analytic Rules/TomcatMultipleClientErrorsFromSingleIP.yaml

```diff
@@ -0,0 +1,32 @@
+id: 4fa66058-4870-11ec-81d3-0242ac130003
+name: Tomcat - Multiple client errors from single IP address
+description: |
+  'Detects multiple client errors from one source in short timeframe'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  let threshold = 100;
+  TomcatEvent
+  | where toint(HttpStatusCode) >= 400 and toint(HttpStatusCode) <= 499 

```

Comment:
casting toint() is unnecessary here and decreases perf.  Either have the connector bring the data in as an INT or simply match on the string.

Reply:
Query works in Sentinel w/o casting toint().
Casting toint() was added to pass checks. If there is no casting toint() in files then checks fail with error